app/views/comments/_comment_actions.html.erb

```diff
@@ -6,6 +6,11 @@
   <% end %>
   <% unless comment.unreviewed? %>
     <li><%= link_to ts("Thread"), comment %></li>
+    <% if comment.depth > 0 %>
+    <li>
+      <%= link_to ts("Parent Thread"), comment_url(id: comment.thread) %>
```

Comment:
Does this need to be `comment_url` or can we use `comment_path` to avoid any https/http oddness?

Reply:
Oh, sure. I don't know why I had that backwards, because it did seem very strange to me, but this makes more sense.